shell/.meteor/versions

```diff
@@ -39,7 +39,7 @@ iron:middleware-stack@1.0.9
 iron:router@1.0.9
 iron:url@1.0.9
 jparker:crypto-core@0.1.0
-jparker:crypto-md5@0.1.1
+jparker:crypto-sha256@0.1.1
```

Comment:
I decided the identicon algorithm doesn't need to be perfectly consistent with github because we'll always fetch github users' identicons from actual github. So it now is based on the user ID that we pass into apps, which is sha-256 based. This allows apps to use identicon.js themselves to generate consistent identicons.

Eliminating MD5 was a nice side-effect. (Previously I needed it because that's what Github uses.)